tests/functional/generate_pages.py

```diff
@@ -69,6 +69,13 @@
 
 """
 
+RELATIONSHIP_RADIO_CLICKER = r"""  click{optionName}() {
+    browser.element('[id="{optionId}"]').click().pause(300)
```

Comment:
Rather than an arbitrary pause duration, is there some way we could wait until something in the DOM changes? e.g. a CSS class gets added? e.g. `browser.waitForExist(selector...`